kong/plugins/key-auth/handler.lua

```diff
@@ -169,6 +162,12 @@ local function do_authentication(conf)
 end
 
 
+local KeyAuthHandler = {
```

Comment:
Same here.

Reply:
Usually in coding, you want variables close to usage, but I can bend on this.